.github/workflows/production-build.yaml

```diff
@@ -1,11 +1,20 @@
 name: Unstract Docker Image Build and Push (Production)
 
 on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Docker image tag"
+        required: true
+      branch:
+        description: "Branch to be used"
+        required: true
```

Comment:
@jaseemjaskp @johnyrahul is this actually needed? I believe branch will anyway be asked as part of the workflow run right?

Reply:
Check the screenshot. Here in this case branch shouldn't be needed. It will  be duplication.
![image](https://github.com/Zipstack/unstract/assets/100672805/8b0b75f7-a90e-4114-b743-f8cac19dd019)

Reply:
I had the same question. It refers to branch  where the workflow file is used. if I havent mentioned it will always take from main.

Reply:
Removed.